### ISSUE TITLE
Instructions say "run your app using `dart bin/__projectName__.dart`"

### ISSUE THREAD

**Issue by DanTup** [dart-lang/stagehand#672]
After creating an app, it says:

```
6 files written.

--> to provision required packages, run 'pub get'
--> run your app using `dart bin/__projectName__.dart`.
```

It looks a bit weird having `__projectName__` here - should it have been replaced?

**Comment by devoncarew** [comment]
Yes, this is a bug. The text doesn't participate in the template replacement, so likely shouldn't have been updated as part of https://github.com/dart-lang/stagehand/pull/659.

We could do something like replace the guidance with `'dart run'`.

cc @jwren
